services/manifest-repo-export-service/pkg/cmd/server.go

```diff
@@ -101,6 +102,15 @@ func Run(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	enableMetricsString, err := readEnvVar("KUBERPULT_ENABLE_METRICS")
+	if err != nil {
+		enableMetricsString = "false"
```

Comment:
please return an error (and exit) if an env var is not set.
Reason:
We have 2 places with default values: the helm chart (values.yaml) and the code here.
There should only be one place.